chainer_chemistry/dataset/parsers/csv_file_parser.py

```diff
@@ -184,5 +172,19 @@ def parse(self, filepath, return_smiles=False, target_index=None):
             return {"dataset": NumpyTupleDataset(result), "smiles": smileses}
 
     def extract_total_num(self, filepath):
-        df = _read_csv(filepath)
+        """Extracts total number of data which can be parsed
+
+        This method is to determine the value fed to `target_index` option of
```

Comment:
I would suggest to rephrase as "We can use this method to determine the value fed to ..." because the usage described here is not the only way to use this method (user may use this method in different way).